### ISSUE TITLE
Horizontal scrollbar always appears when not using original-size

### ISSUE THREAD

**Issue by conner-fallone** [VadimDez/ng2-pdf-viewer#1036]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```

The horizontal scrollbar always seems to appear when you are not using [original-size]="true". It's noticeable even on the demo page here: https://vadimdez.github.io/ng2-pdf-viewer/. In doing some research, it looks like the issue is simply that the page width (including the canvasWrapper and textLayer) that gets assigned is too big, and goes underneath the vertical scrollbar slightly, resulting in the horizontal scrollbar appearing.

**Comment by Hely0n** [comment]
That's true.  A workaround I found is to invoke "window.dispatchEvent(new Event('resize'))" after rendering.

**Comment by PeterBenko** [comment]
I assume this hasn't been fixed yet

**Comment by BohdanBida** [comment]
After researching the source code I found a solution for myself

This issue can be resolved by calling the `updateSize()` method of the `PdfViewerComponent` class on the `pages-initialized` event.

Example
component.html:
```
<pdf-viewer [src]="src" [original-size]="false" [rotation]="0" [show-all]="true"[zoom]="zoom" [zoom-scale]="'page-width' [render-text]="true" [external-link-target]="'blank'" [page]="page" (pages-initialized)="onInitialize($event)">
</pdf-viewer>
```

component.ts:
```
@ViewChild(PdfViewerComponent) public pdfViewerComponent: PdfViewerComponent;

public onInitialize(): void {
        this.pdfViewerComponent.updateSize();
}
```

**Comment by baldmaan** [comment]
@BohdanBida solution works great, thanks! Only will mention that onInitialize() should be without an `$event`
`<pdf-viewer (pages-initialized)="onInitialize()"></pdf-viewer>`
